# Say a sentence

string message
---
bool result